### PR TITLE
PS-9161 Update APT and YUM topics with more explanation - 8.0

### PR DESCRIPTION
**Pull request by patrickbirch**
On branch ps-9161-8
	modified:   docs/apt-download-deb.md

### CODE REVIEW

File: docs/apt-download-deb.md [hunk 1]
```diff
@@ -2,44 +2,66 @@
...
+        percona-server-test_8.0.31-23-1.buster_amd64.deb
+        ```
+
+3. Install Percona Server for MySQL using `dpkg`. Run this command, that uses the `dpkg` command to install Debian (.deb) packages, as root or use the sudo command. `sudo` is a command that allows you to run programs with the security privileges of another user, usually as the superuser. Installing packages typically requires superuser or 'root' privileges.
+
+    `dpkg` is a package manager for Debian-based systems and can install, remove, and provide information about `.deb` packages. 
+
+    The `-i` option tells `dpkg` to install the package.
+
+    The `*.deb` is a wildcard that matches any file in the current directory that ends with the `.deb` extension.
```
Comment (alina-derkach-oaza) [inline docs/apt-download-deb.md:58]: Add bullets to highlight the options

File: docs/tokudb-version-changes.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+We have removed the storage engine from the installation packages and disabled the storage engine in our binary builds.
+
+## Disabled 
+
+[Percona Server for MySQL 8.0.26-16](release-notes/Percona-Server-8.0.26-16.md) includes the  TokuDB storage engine plugins in the binary builds and packages but disables them.
+
+The `tokudb_enabled` option and the `tokudb_backup_enabled` option control the state of the plugins and have a default setting of `FALSE`. The plugins fail to initialize and print a deprecation message if you attempt to load them.
+
+We recommend [migrating the data to the MyRocks storage engine](https://docs.percona.com/percona-server/latest/tokudb/removing_tokudb.html#migrate-myrocks).
```
Comment (alina-derkach-oaza) [inline docs/tokudb-version-changes.md]: The link does not work

File: docs/yum-download-rpm.md [hunk 3]
```diff
@@ -1,18 +1,32 @@
...
 Download the packages from [Percona Product Downloads](https://www.percona.com/downloads). If needed, [Instructions for the Percona Product Download](download-instructions.md) are available.
 
  The following example downloads *Percona Server for MySQL* 8.0.29-21 release packages for *RHEL* 8.
 
-1. Use `wget` to download the tar file.
+Using `Wget`, the following command downloads a specific version of the Percona Server for Red Hat Enterprise Linux 8 from the Percona website. 
 
 	```{.bash data-prompt="$"}
 	$ wget https://downloads.percona.com/downloads/Percona-Server-LATEST/Percona-Server-8.0.29-21/binary/redhat/8/x86_64/Percona-Server-8.0.29-21-rc59f87d2854-el8-x86_64-bundle.tar
 	```
```
Comment (alina-derkach-oaza) [inline docs/yum-download-rpm.md:15]: This command is broken on preview

File: docs/yum-run.md [hunk 4]
```diff
@@ -3,38 +3,36 @@
...
 
-3. Stop the service with the following command:
-	
 	```{.bash data-prompt="$"}
-	$ sudo service mysql stop
+	$ sudo systemctl start mysql
+	$ sudo systemctl status mysql
+	$ sudo systemctl stop mysql
+	$ sudo systemctl restart mysql
 	```
```
Comment (alina-derkach-oaza) [inline docs/yum-run.md]: This command is broken on the preview

File: docs/yum-download-rpm.md (hunk 3 above)
Comment (patrickbirch) [reply docs/yum-download-rpm.md:15]: Do you mean the wget command? It works in Docker.

File: docs/yum-run.md (hunk 4 above)
Comment (patrickbirch) [reply docs/yum-run.md]: It works in Docker.

File: docs/yum-download-rpm.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/yum-download-rpm.md:15]: https://percona-server-docs-8-0-pr-296.onrender.com/yum-download-rpm.html

File: docs/yum-run.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/yum-run.md]: It's broken on Render preview https://percona-server-docs-8-0-pr-296.onrender.com/yum-run.html#manage-the-service

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**